NextJS integration (#23)
fingerprintjs/fingerprintjs-pro-react: contributing.md

### Development playground

> It's a 2 demo page for this integration:

Reviewer: How about:

```suggestion
There are 2 demo pages for this integration:
```

?
